client/reader/following/style.scss

```diff
@@ -73,9 +73,19 @@
 	}
 }
 
-.following-stream__quick-post-card.card.is-expanded {
-	.foldable-card__content {
-		overflow: visible;
+.following-stream__quick-post-card.card {
+
+	.foldable-card__action:focus {
+		outline: none;
+		border-color: var(--color-primary);
+		box-shadow: 0 0 0 2px var(--color-primary-light);
+		border-radius: 2px;
```

Comment:
This would fit better with a 6px radius, but the mix of border-radius in the Reader is a little confusing to me. Maybe the other borders should be 2px instead. p1741726788870729-slack-C083J8DHLLD

<img width="346" alt="Screenshot 2025-03-11 at 4 02 18 PM" src="https://github.com/user-attachments/assets/93e4fe63-1541-41aa-a39e-b191deb1c261" />

Reply:
Noting we should look for `accessible-focus` class when applying these focus styles for a11y flows: e.g. https://github.com/Automattic/wp-calypso/pull/101240